app/helpers/searchHelpers.php

```diff
@@ -432,3 +432,166 @@ function caGetChangeLogForElasticSearch($po_db, $pn_table_num, $pn_row_id) {
 
 		return $va_return;
 	}
+	# ---------------------------------------
+	function caGetQueryBuilderFilters(BaseModel $t_subject, Configuration $vo_query_builder_config) {
+		$vs_table = $t_subject->tableName();
+		$t_search_form = new ca_search_forms();
+		$va_filters = array_values(array_map(
+			function ($pa_bundle) use ($t_subject, $vo_query_builder_config) {
+				return caMapBundleToQueryBuilderFilterDefinition($t_subject, $pa_bundle, $vo_query_builder_config);
+			},
+			$t_search_form->getAvailableBundles($vs_table)
+		));
+		$va_exclude = $vo_query_builder_config->get('query_builder_exclude_' . $vs_table);
+		$va_filters = array_filter($va_filters, function ($vo_filter) use ($va_exclude) {
+			return array_search($vo_filter['id'], $va_exclude) === false;
+		});
+		$va_priority = $vo_query_builder_config->get('query_builder_priority_' . $vs_table);
+		usort($va_filters, function ($po_a, $po_b) use ($va_priority, $vs_table) {
```

Comment:
`$pa_a` and `$pa_b`, as you're accessing it like an array (and I don't think I've seen `ArrayAccess` in the CA codebase ;)